e2e-tests/cypress/tests/support/external_commands.ts

```diff
@@ -13,7 +13,7 @@ import {getRandomId} from '../utils';
 function externalActivateUser(userId: string, active = true) {
     const admin = getAdminAccount();
 
-    cy.externalRequest({user: admin, method: 'PUT', path: `users/${userId}/active`, data: {active}});
+    return cy.externalRequest({user: admin, method: 'PUT', path: `users/${userId}/active`, data: {active}});
```

Comment:
any special reason for this return here?

Reply:
So the function does not break the functionality in `channels/enterprise/guest_accounts/guest_identification_ui_not_cloud_spec.ts`

```typescript
// # Deactivate Guest user
cy.externalActivateUser(guestUser.id, false).wait(TIMEOUTS.FIVE_SEC); // .wait
```